### ISSUE TITLE
Accessibility: Correct missing/orphaned form labels and buttons

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-reporting#662]
Form labels are missing from many non-text data entry fields like checkboxes, drop-down menus and
other fields. There are also orphan labels that are no longer needed that should be removed

## **Industry interface**

**Administration pages:**
- Operations
   - [ ] Missing form labels for the search boxes

 - View Details on Operations
   - [ ] 14 orphaned form labels - all labels

 - View Facility
   - [ ] 9 orphaned form labels - all labels

 - Contacts
   - [ ] Missing form labels for search boxes

 - View Details on Contacts
   - [ ] 10 orphaned form labels - all labels

**Register an operation form**
 - Operation information page 
   - [ ] 2 orphaned labels (review & remove/update as needed) - heading & `Select Operation Type` field
 - Facility information page: 
   - [ ] 4 orphaned labels (review & remove/update as needed) - heading, `Facility Name` label, `Facility Type` label, `Province` label
   - [ ] Missing form labels for the search boxes

 - Facility information (for specific LFO facility)
   - [ ] 11 orphaned labels (review & remove/update as needed)

 - New Entrant Operation page
   - [ ] Missing form label for Upload Attachment
   - [ ] 2 orphaned form labels - heading & `When is this operation’s date of First Shipment?*` label

 - Operation information page 
   - [ ] 1 orphaned label (review & remove/update as needed)

 - Submission page 
   - [ ] 1 orphaned label (review & remove/update as needed)

**Reporting**

 **Submit a report page**
  - [ ] 3 missing form labels (add)
  - [ ] 11 empty buttons (put content in the button element or give the input a value)

**Operation information**
 - Review operation information
   - [ ] 1 empty form label - `Select what type of report you are filling. If you are uncertain about which report type your operation should complete, please contact GHGRegulator@gov.bc.ca.` label
   - [ ] 1 orphaned form label - heading
 
 - Person responsible
   - [ ] 1 empty form label - `Select contact if they are already a BCIERS user` label
   - [ ] 1 orphaned form label - heading

 - Review facilities
   - [ ] 1 orphaned form label - heading

**Report information (LFO facilities list)**
- [ ] 2 missing form labels on search boxes

**Facility information**
 - Review facility information
   - [ ] 1 empty form label - `Facility type` field
   - [ ] 1 orphaned form label - heading

 - Activities information (with both boxes selected)
   - [ ] 2 missing form labels - on source type checkboxes
   - [ ] 6 empty form labels
   - [ ] 2 empty buttons
   - [ ] 3 orphaned form labels - heading, source type labels
 - Non-attributable emissions
   - [ ] 4 orphaned form labels - heading, `Did your non-attributable emissions exceed 100 tCO2e?` label, `Select Emissions Category`, `Gas Type`
 - Emissions summary
   - [ ] 4 orphaned form labels - heading, subheadings
 - Production data
   - [ ] 1 orphaned form label

**General stationary combustion excluding line tracing**
- [ ] 2 missing form label
- [ ] 4 empty form label
- [ ] 1 empty button
- [ ] 3 orphaned form labels

**Fuel combustion by mobile equipment**
- [ ] 2 empty form labels
- [ ] 1 empty button
- [ ] 1 orphaned form label

Production data
- [ ] 2 empty form labels 
- [ ] 2 broken ARIA references (fix target reference)
- [ ] 3 orphaned labels

**Allocation of emissions**
- [ ] 1 empty form label
- [ ] 27 orphaned form labels

**Additional reporting data**
- [ ] 2 orphaned form labels - heading, `Did you capture emissions?` label

**New Entrant Information**
- [ ] 2 orphaned form labels - heading, `Unit` label

**Operation Emission Summary**
- [ ] 4 orphaned form label - heading, subheadings

**Compliance summary**
- [ ] 2 orphaned form labels - heading, subheading

**Review operation information**
- [ ] 2 missing form labels
- [ ] 1 empty form label
- [ ] 2 broken AIRA references
- [ ] 148 orphaned form labels

**Verification** 
- [ ] 3 orphaned form labels - heading, `Select accrediting body`, `Were there any threats to independence noted`

**Attachments**
- [ ] Missing page title - #648 
- [ ] 4 empty form labels - all upload attachments

**Sign-off**
- [ ] 2 Orphaned form labels - header, `Date signed`

**Compliance**
- [ ] 7 missing form labels - all search fields
___
## CAS interface

**Administration > Operators**
- [ ] 4 missing form labels - all search fields

**Administration > Operations**
- [ ] 5 missing form labels - all search fields

**Administration > Operator Administrators And Access Requests**
- [ ] 7 missing form labels - all search fields

**Transfers**
- [ ] 3 missing form labels - all search fields

**View Details on Transfers**
- [ ] 6 orphaned form labels - headings, all labels

#### Acceptance Criteria

- [ ] Download the WAVE plugin from the Chrome plugin store
- [ ] Run the extension on all pages mentioned (+ any new ones not mentioned here) to see where issues are
- [ ] Update coding where identified above

#### Additional context
Ensure that all parts of the webpage have form and AIRA labels if no text-based labels and remove orphan form
labels where no longer needed.

Related WCAG Guidelines
Level AA 1.3.5 Clearly define the purpose of input fields: Give people clear labels and instructions so
they know how and what to enter in input fields. This includes AIRA labels.

**Comment by NicoleGovvy** [comment]
@fviduya I've created a ticket for the form labels needing review - if you had a few minutes could you look this over and see if it reads ok for a dev looking at it? Thank you!

**Comment by fviduya** [comment]
Hi @NicoleGovvy, I think it looks good, you got most of it, thank you! I've updated the description and added areas that were not included before, I also added details on some of them 🙂
